src/main.rs

```diff
@@ -220,32 +238,85 @@ impl FuzzProject {
            .arg("run")
            .arg("--manifest-path")
            .arg(self.manifest_path());
-        if release {
+        if !args.is_present("no_opt") {
             cmd.arg("--release");
         }
         cmd.arg("--verbose")
            .arg("--bin")
            .arg(&target)
            .arg("--target")
-           // won't pass rustflags to build scripts
-           .arg(target_triple)
+           .arg(utils::default_target()) // won't pass rustflags to build scripts
```

Comment:
> The thing is, there's actually no way to pass in triple target right now anyways because the corresponding flag doesn't have a takes_value(true) (see https://github.com/rust-fuzz/cargo-fuzz/blob/master/src/main.rs#L57) 😸!

This is a bug that was fixed in https://github.com/rust-fuzz/cargo-fuzz/pull/109 which I simply forgot to merge before today.